src/mudlet-lua/lua/GUIUtils.lua

```diff
@@ -623,9 +623,9 @@ end
 ---   <pre>
 ---   deselect("myMiniConsole")
 ---   </pre>
-function deselect()
```

Comment:
Promote Issue https://github.com/vadi2/mudlet-lua/issues/15 to a PR I take it...?